### PR TITLE
Add runtime env sql&models

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: pkg/db/schema/runtime/V0_1__init.sql.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	description     TEXT                                NOT NULL,
+	runtime_env_url TEXT                                NOT NULL,
+	owner           VARCHAR(50)                         NOT NULL,
+	status          VARCHAR(50)                         NOT NULL,
+	create_time     TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+	status_time     TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+CREATE TABLE runtime_env_label (
+	runtime_env_id VARCHAR(50) NOT NULL,
```
Comment (zheng1) [inline pkg/db/schema/runtime/V0_1__init.sql.sql]: There should be used `runtime_env_label_id` as PRIMARY KEY.

Comment (runzexia) [reply pkg/db/schema/runtime/V0_1__init.sql.sql]: If I use runtime_env_id + Label_key as PRIMARY key is reasonable ?

File: pkg/db/schema/runtime/V0_1__init.sql.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
+CREATE TABLE runtime_env (
```
Comment (chai2010) [inline pkg/db/schema/runtime/V0_1__init.sql.sql]: while the file name is `x.sql.sql`, double `.sql`?

File: pkg/models/runtime_env_credential.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+func RuntimeEnvCredentialContentStringToMap(stringContent string) map[string]string {
+	var mapContent map[string]string
+	json.Unmarshal([]byte(stringContent), &mapContent)
+	return mapContent
+}
+
+func RuntimeEnvCredentialContentMapToString(mapContent map[string]string) string {
+	stringContent, _ := json.Marshal(mapContent)
```
Comment (chai2010) [inline pkg/models/runtime_env_credential.go]: pls do not ignore any error!
return err, or log error, or panic.

File: pkg/models/runtime_env_label.go [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+const RuntimeEnvLabelTableName = "runtime_env_label"
+
+func NewRuntimeEnvLabelId() string {
+	return utils.GetUuid("rel-")
+}
+
+type RuntimeEnvLabel struct {
+	RuntimeEnvLabelId string
```
Comment (zheng1) [inline pkg/models/runtime_env_label.go]: The code is not formatted. Please `git commit` after `make fmt-all`

Comment (runzexia) [reply pkg/models/runtime_env_label.go]: done

File: pkg/models/runtime_env_credential.go [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+func RuntimeEnvCredentialToPbs(runtimeEnvCredentials []*RuntimeEnvCredential) (pbRuntimeEnvCredentials []*pb.RuntimeEnvCredential) {
+	for _, runtimeEnvCredential := range runtimeEnvCredentials {
+		pbRuntimeEnvCredentials = append(pbRuntimeEnvCredentials, RuntimeEnvCredentialToPb(runtimeEnvCredential))
+	}
+	return
+}
+
+func RuntimeEnvCredentialContentStringToMap(stringContent string) map[string]string {
+	var mapContent map[string]string
+	json.Unmarshal([]byte(stringContent), &mapContent)
```
Comment (zheng1) [inline pkg/models/runtime_env_credential.go]: The error should not ignore

File: pkg/models/runtime_env_label.go [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+func NewRuntimeEnvLabel(runtimeEnvId, labelKey, labelValue string) *RuntimeEnvLabel {
+	return &RuntimeEnvLabel{
+		RuntimeEnvLabelId: NewRuntimeEnvLabelId(),
+		RuntimeEnvId:      runtimeEnvId,
+		LabelKey:          labelKey,
+		LabelValue:        labelValue,
+	}
+}
+
+func RuntimeEnvLabelToString(runtimeEnvLabel *RuntimeEnvLabel) string {
```
Comment (zheng1) [inline pkg/models/runtime_env_label.go]: runtime_env_label should return as a struct.
reference: https://github.com/runzexia/openpitrix/commit/a9a1fdbb97d7d92d1af526f58dab42b1cb86087b#r28152438

**Comment by zheng1** [comment]
LGTM, Please squash the git commits under the master branch.

**Comment by runzexia** [comment]
done

**Pull request merged**